### PR TITLE
Stable 2.0 backport

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by joakim-tjernlund** [comment]
next stable is becoming very large , any ETA on next release?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+* CMake option WITH_PROXY_MODULES is currently experimental, do not use in
+production.
+
+Noteworthy changes:
+* Progressive codec improvements. Reduces graphical glitches against windows
+and ogon servers
+* Fixed +glyph-cache, now working properly without disconnects
+* Huge file support in clipboard
+* XWayland support for xfreerdp (keyboard grabbing)
+* Imporved wlfreerdp (wayland client)
```
Comment (Fantu) [inline ChangeLog:13]: @akallabeth small typo: Imporved->Improved

Comment (akallabeth) [reply ChangeLog:13]: @Fantu thank you, missed that one ^^

**Comment by SnejPro** [comment]
Causes crashing while connecting with /smartcard when using Datev MiDentity (ref: https://github.com/FreeRDP/FreeRDP/issues/6777#issuecomment-773965143)

**Comment by SnejPro** [comment]
> 
> 
> Causes crashing while connecting with /smartcard when using Datev MiDentity (ref: [#6777 (comment)](https://github.com/FreeRDP/FreeRDP/issues/6777#issuecomment-773965143))

Now everything works :)

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by joakim-tjernlund** [comment]
@mfleisz , please review so this PR can be merged.

**Comment by akallabeth** [comment]
@sunweaver we´ve (finally) decided to merge this. A `2.3.0` tag has been created matching the next release.
Release mail etc may be delayed until next week due to being short on time.

**Comment by sunweaver** [comment]
On  Do 25 Feb 2021 09:58:43 CET, akallabeth wrote:

> @sunweaver we´ve (finally) decided to merge this. A `2.3.0` tag has  
> been created matching the next release.
> Release mail etc may be delayed until next week due to being short on time.

Awesome, will shove it into Debian 11 this afternoon.

Mike
-- 

DAS-NETZWERKTEAM
c\o Technik- und Ökologiezentrum Eckernförde
Mike Gabriel, Marienthaler Str. 17, 24340 Eckernförde
mobile: +49 (1520) 1976 148
landline: +49 (4351) 850 8940

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

**Pull request merged**